rllib/algorithms/ppo/torch/ppo_torch_rl_module.py

```diff
@@ -0,0 +1,11 @@
+from ray.rllib.core.examples.simple_ppo_rl_module import (
+    SimplePPOModule,
+    FCConfig,
+    PPOModuleConfig,
+)
+
+__all__ = [
+    "SimplePPOModule",
```

Comment:
Should we just move SimplePPOModule into this file already? Everything seems a little dispersed right now and I would like to start moving things into their right place. Or at least try :)